mailgreen/services/carbon_service.py

```
@@ -0,0 +1,110 @@
+import logging
+from datetime import datetime, timedelta, timezone
+from collections import defaultdict
```

Ruff (0.11.9)

`collections.defaultdict` imported but unused

Remove unused import: `collections.defaultdict`

(F401)